packages/apollo-server/README.md

```diff
@@ -6,30 +6,36 @@
 [![Join the community on Spectrum](https://withspectrum.github.io/badge/badge.svg)](https://spectrum.chat/apollo)
 
 
-Apollo Server is a community-maintained open-source GraphQL server. It works with pretty much all Node.js HTTP server frameworks, and we're happy to take PRs for more! Apollo Server works with any GraphQL schema built with [GraphQL.js](https://github.com/graphql/graphql-js), so you can build your schema with that directly or with a convenience library such as [graphql-tools](https://www.apollographql.com/docs/graphql-tools/).
+Apollo Server is a community-maintained open-source GraphQL server. It works with pretty much all Node.js HTTP server frameworks, and we're happy to take PRs to add more! Apollo Server works with any GraphQL schema built with [GraphQL.js](https://github.com/graphql/graphql-js)--so you may build your schema with that or a convenience library such as [graphql-tools](https://www.apollographql.com/docs/graphql-tools/).
 
 ## Principles
 
 Apollo Server is built with the following principles in mind:
 
-- **By the community, for the community**: Apollo Server's development is driven by the needs of developers.
-- **Simplicity**: By keeping things simple, Apollo Server is easier to use, easier to contribute to, and more secure.
-- **Performance**: Apollo Server is well-tested and production-ready.
+- **By the community, for the community**: Its development is driven by the needs of developers.
+- **Simplicity**: By keeping things simple, it is more secure, easier to use, and easier to contribute to.
+- **Performance**: It is well-tested and production-ready.
```

Comment:
These are all great changes!  If you could find a way to remove the `easier` references here (and even the `easy` below as well), it would be nice to de-emphasize the determination of effort, since that may very well vary substantially from user to user.

Reply:
How about the following?

**"Simplicity: By keeping things simple, it is more secure and easier to implement and contribute to."**

Essentially the **simplicity** principle _is_ subjective on its own.

Reply:
I like it!  I've added it as a code suggestion above.